app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -26,8 +26,14 @@ def index
           else
             response = proofer.request_pilot_facilities
           end
-        rescue => err
-          Rails.logger.warn(err)
+        rescue => error
+          Rails.logger.warn(
+            {
+              error_name: error.class.name,
+              message: error.message,
+              source: self.class.name,
+            }.to_json,
```

Comment:
I'd also add a descriptive key/event name so we can have something to find these by (not sure it's the best name, but the general idea)
```suggestion
            {
              warning: "UspsLocationsController#index argcis"
              error_name: error.class.name,
              message: error.message,
              source: self.class.name,
            }.to_json,
```